packages/react/src/utils/inertValue.ts

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+
+export function inertValue(value?: boolean): string | boolean | undefined {
+  const pieces = React.version.split('.');
+  const major = parseInt(pieces[0], 10);
+  if (major >= 19) {
+    return value;
+  }
+  // compatibility with React < 19
+  return value ? 'true' : undefined;
+}
```

Comment:
There's already a `reactVersion.ts` util (`isReactVersionAtLeast`)

Reply:
OK, let me go and update this!

Reply:
There's a type error curently, you can just cast the fallback to boolean to silence it

Reply:
Sorry, I made assumptions that what I was inlining from `react-spectrum` didn't have bad types...!